restapi.go

```diff
@@ -874,6 +874,79 @@ func (s *Session) GuildRoleDelete(guildID, roleID string) (err error) {
 	return
 }
 
+// GuildPruneCount Returns the number of members that would be removed in a prune operation.
+// Requires 'KICK_MEMBER' permission.
+// guildID	: The ID of a Guild.
+// days		: The number of days to count prune for (1 or more).
+func (s *Session) GuildPruneCount(guildID string, days uint32) (count uint32, err error) {
+	count = 0
+
+	if days <= 0 {
+		err = errors.New("The number of days should be more than or equal to 1.")
+		return
+	}
+
+	u, err := url.Parse(EndpointGuildPrune(guildID))
+	if err != nil {
+		return
+	}
+
+	q := u.Query()
```

Comment:
Should be fine just to do:

uri := EndpointGuildPrune(guildID) + "?days="+days